cookbook/form/data_transformers.rst

```diff
@@ -106,10 +97,10 @@ in your code.
 You can also add the transformer, right when adding the field by changing the format
 slightly::
 
-    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
 
     $builder->add(
-        $builder->create('description', TextareaType::class)
+        $builder->create('description', TextType::class)
```

Comment:
Since text type is the default, I would suggest to remove the second argument everywhere to easy the merge in all branches.

Why not also add a tip to handle `null` as an empty string, we could then remove the hint in master and make a PR to document it in `empty_data` option for the text type (ref symfony/symfony#18357)?